submissions/OlexiyDobroskok/OOP_exercise/index.js

```diff
@@ -0,0 +1,138 @@
+class Inhabitant {
+  constructor({
+    species = "No species",
+    name = "No name",
+    gender = "No gender",
+    saying = "Hello",
+    friends = [],
+  }) {
+    this.species = species;
+    this.name = name;
+    this.gender = gender;
+    this.saying = saying;
+    this.friends = friends;
+  }
+
+  addFriend(person) {
+    this.friends.push(person);
+  }
+
+  introduceYourSelf() {
+    if (this.friends.length === 0) {
+      this.friends = "Looking for friends!";
```

Comment:
What will happen if we `.addFriend` after first call of `.introduceYourSelf`?

Reply:
Instead of an empty array, I get a string...)

Reply:
@

> What will happen if we `.addFriend` after first call of `.introduceYourSelf`?

@OleksiyRudenko Instead of an empty array, I get a string...) Done)